tables/de-g0-bd.utb

```diff
@@ -0,0 +1,91 @@
+# liblouis: German Grade 0 Braille (bidirectional
+#
+# Das System der deutschen Blindenschrift (2015/2018)
+# http://www.bskdl.org/textschrift.html
+#
+# Bemerkung:
+# - Gross-/Kleinschreibung wird nicht berücksichtigt (d.h. für
+#   Abkürzungen u.ä. braucht es Vorprogramm)
+# - Akzente: Hinweis auf die beiden Tabellen (bis jetzt nur als
+#   Kommentar in de-g0.utb)
+#
+# -----------
+# Differences between this table (de-g0-bd.utb) and de-g0.utb
+# This table is meant to be used with screen readers and note-takers
+# for both forward and backward translation.
+# Forward translation in this table is designed to give as much
+# information as possible in order for the result to be
+# back-translatable.
+# Forward translation differs from that of de-g0.utb in the following ways.
+# 1. All capital letters are marked.
+# 2. Accented letters are translated using de-accents-detailed.cti
+#    to make the translation as detailed as possible.
+# 3. "`" and "´" are translated as "'" so they won't cover other
+#    characters.
+# -----------
+
+#-name: Deutsche Basisschrift
+#-display-name: German uncontracted braille
```

Comment:
This metadata needs to be updated.

Reply:
> Then, what do you want instead? Should I just add (bidi)?

I'm not sure what it should be. You tell me :-) No, just adding "bidi" is way to criptic. It needs to be something understandable. Maybe "German uncontracted braille, back-translatable"?

Reply:
> How about simply “bidirectional”, which is what “bidi” stands for anyway?
>
> I think we need some “Liblouis terms” to describe these things properly. With the Danish tables, the bidirectional ones came first, so “lit” was appended to the other ones, which is also a bit cryptic.

Between "back-translatable" and "bidirectional", I find the former more to the point and it's more understandable language. But both mentioned options focus on the transcription process rather than the writing system itself. Maybe that is inevitable.

Yes, we need a good system-wide term for this category of tables. For Danish, the tables with "lit" and without "lit" actually have exactly the same display name at the moment, which is not good. I don't care about the file names of the tables, they are computer things so are allowed to be a bit cryptic. But the stuff that is visible to the user, like the `display-name` and `index-name` metadata fields, should be as clear as possible, and consistent across tables.

Reply:
I updated the metadata. I haven't changed the metadata of the Danish tables yet.

Reply:
It's too late to change it now, but I think the term "detailed" summarizes well what this braille system variant is.